packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedFieldMessageEmbed.ts

```diff
@@ -0,0 +1,155 @@
+import { MessageEmbed, MessageEmbedOptions } from 'discord.js';
+
+import { PaginatedMessage } from './PaginatedMessage';
+
+/**
+ * This is a utility of {@link PaginatedMessage}, but paginating only the field of an embed.
+ * You must either use this class directly or extend it.
+ *
+ * @example
+ * ```typescript
+ * const { PaginationFieldMessageEmbed } = require('@sapphire/discord.js-utilities');
+ *
+ * new PaginationFieldManager()
+ *    .setTitleField('')
+ *    .setTemplate({ ...options })
+ *    .setItems([{ field: 'value' }, { field: 'value' }, { field: 'value' }, { field: 'value' }])
+ *    .setItemsPerPage(2)
+ *    .make()
```

Comment:
Why is a `.make()` call required?

Reply:
Because this method generates the pages according to the number of items in the array.

Reply:
Ah ok